perf/renaissance/playlist.xml

```diff
@@ -94,7 +94,7 @@
 				<platform>x86-32_windows|arm_linux</platform>
 			</disable>
 		</disables>
-		<command>$(JAVA_COMMAND) --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.util.concurrent=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED  -jar $(Q)$(TEST_RESROOT)$(D)renaissance.jar$(Q) --json $(Q)$(REPORTDIR)$(D)dec-tree.json$(Q) dec-tree; \
+		<command>$(JAVA_COMMAND) -jar $(Q)$(TEST_RESROOT)$(D)renaissance.jar$(Q) --json $(Q)$(REPORTDIR)$(D)dec-tree.json$(Q) dec-tree; \
```

Comment:
`$(ADD_OPENS_CMD)` is missing?

Reply:
yea wow okay is like i omitted that

Reply:
no changes ... same error as before. 
 grinder: https://ci.adoptopenjdk.net/job/Grinder/3035/console